api/app/utils/time.py

```diff
@@ -120,3 +120,15 @@ def get_utc_datetime(input_datetime: datetime):
 
 def get_days_from_range(start_time: datetime, end_time: datetime) -> List[datetime]:
     return [start_time + timedelta(days=x) for x in range((end_time - start_time).days + 1)]
+
+
+def get_model_prune_range(now: datetime, start: datetime, end: datetime):
+    # only allow pruning up until 19 days before today
+    max_end = now - timedelta(days=19)
```

Comment:
`machine_learning.py` also uses the number 19, could we define a constant somewhere or set things up to be read from a config map? Maybe suitable as a follow on ticket?

Reply:
Should we consider defaulting to 21 days to be safe and consistent with `apply_data_retention_policy()` in `common_model_fetchers.py`?

Reply:
I pulled out the 21 day timedelta as constant and used it across this job and `apply_data_retention_policy()` in: https://github.com/bcgov/wps/pull/2952/commits/69c5fa021894c13b8ff7eaf111fcc151ab7c6bfd